velox/functions/FunctionRegistry.h

```diff
@@ -95,4 +97,24 @@ resolveVectorFunctionWithMetadata(
 /// Clears the function registry.
 void clearFunctionRegistry();
 
+/// A function name is a companion function's if the name is an existing
```

Comment:
Yes this was subsequently made external because it is also used in `CoverageUtil.cpp`.

Reply:
Do you need it right now ? If not, then lets not expose it. We can do so later when we are working on the CoverageUtil PRs.

Reply:
Lets move it back to CoverageUtil.h

Reply:
Moved it back to CoverageUtil, but now `velox_function_registry` would have a dependency on `velox_coverage_util`, and this results in a circular dependency between them. It doesn't appear to be causing any issues but I would prefer to avoid this. Could you please share if this is fine?

Reply:
That's okay. We can keep it around here.